RedHatInsights/sed-frontend
# fix(RHINENG-3363): Alter permissions for change settings button

Fixes https://issues.redhat.com/browse/RHINENG-3363, https://issues.redhat.com/browse/RHINENG-3362.

This changes the required inventory permission to inventory:hosts read and write.

# Description

'inventory:\*:read' is no longer available for the administrator roles. This was causing the change settings button to be always disabled regardless user's roles. Since "RHC Administrator" and "Inventory Hosts Administrator" are the required roles, we are turning 'inventory:\*:read' into 'inventory:hosts:read' and 'inventory:hosts:write'.

# How to test the PR

1. Have an account with RHC Administrator and Inventory Hosts Administrator role,
2. Go to /insights/connector, make sure the page loads and you can click on Change settings.
